src/lib.rs

```diff
@@ -8,6 +8,8 @@ mod utils;
 // Helper Methods
 pub use targets::get_all_targets;
 pub use targets::Target;
+pub use targets::Window;
```

Comment:
These are the only changes from me, I checked out from a branch that was supposed to have scale factor changes, Hence the diff is kinda...